src/shared/internal-calendar/internal-calendar-month.style.tsx

```diff
@@ -53,16 +53,16 @@ export const MonthCell = styled.div<StyleProps>`
     }
 
     ${(props) => {
-        if (props.$disabled) {
-            return css`
-                cursor: not-allowed;
+        // if (props.$disabled) {
```

Comment:
I might be out of the loop. my understanding is that dates will now be visually disabled but still interactive

if this is confirmed, then the commented code can be removed entirely

also, `handleDayClick` will need to be updated to ignore disabled state